main.js

```diff
@@ -5,17 +5,42 @@ const execa = require("execa");
 
 let win;
 
-function run_program() {
+function run_program(network_type, network_id) {
```

Comment:
@calvinaco could it just take `run_program(args)` and then `await execa("client-rpc", args);` ?

Reply:
possible, but need to catch exception